reana-ui/package.json

```diff
@@ -18,15 +18,14 @@
     "redux": "^4.0.4",
     "redux-devtools-extension": "^2.13.8",
     "redux-thunk": "^2.3.0",
-    "semantic-ui-css": "^2.4.1",
     "semantic-ui-react": "^0.88.2",
     "universal-cookie": "latest"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
+    "start": "craco start",
+    "build": "craco build",
```

Comment:
The build is breaking:
```console
> yarn build                                                                                                         [0/1816]
yarn run v1.21.1                       
$ craco build                          
Creating an optimized production build...                                     
                                       
                    The build failed because the process exited too early.                                                                                   
                    This probably means the system ran out of memory or someone called                                                                       
                    `kill -9` on the process.                                 
                                                                              
error Command failed with exit code 1.                                        
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command. 
```

The build gets killed because the memory limit being reached:
![Screenshot 2020-02-04 at 13 23 05](https://user-images.githubusercontent.com/9035606/73744994-4fcdd000-4752-11ea-9801-9469c9c69c06.png)

Reply:
Tested on my cluster:
* Using `craco`: ~Average: 1.1GiB, Max.: 1.6GiB - Done in 255.62s
* Using `react-scripts`: ~Average: 850MiB, Max.: 975MiB - Done in 98s

Reply:
Pinning `node` version did the trick, thanks @ntarocco for the help!
```diff
-FROM node as react-build
+FROM node:10 as react-build
```

Reply:
Quite interesting! Glad it worked :)